frontend/cypress/pages/DictionaryMenu.js

```diff
@@ -0,0 +1,105 @@
+class DictionaryMenuPage {
+  constructor() {
+    this.selectors = {
+      title: "h2",
+      modal: ".cds--modal-container",
+      add: "[data-cy='addButton']",
+      dictCategory: "#description",
+      dictNumber: "#dictNumber",
+      dictEntry: "#dictEntry",
+      dictValue: "div",
+      notActive: "#downshift-3-item-1",
+      active: "#downshift-3-item-0",
+      isActive: "#isActive",
+      localAbbreviation: "#localAbbrev",
+      searchByDictEntry: "#dictionary-entry-search",
+      cancelButton: ".cds--btn--secondary",
+      updateButton: ".cds--btn--primary",
+      addbutton: "div.cds--btn-set > button.cds--btn--primary",
+      modify: "[data-cy='modifyButton']",
+      deactivate: "[data-cy='deactivateButton']",
+      radioButton: "td.cds--table-column-checkbox.cds--table-column-radio",
+    };
+  }
+
+  verifyPageTitle() {
+    cy.contains(this.selectors.title, "Dictionary Menu");
+  }
+
+  clickAddButton() {
+    cy.get(this.selectors.add).click();
+  }
```

Comment:
thanks @Bahati308 ,,
i see `clickAddButton()` is defined twice and below in this file. If you're aiming to click by text content and data attribute, consolidate ie 
```
clickAddButton() {
  cy.get(this.selectors.add).contains("Add").click();
}
```